karma.conf.maker.js

```diff
@@ -156,12 +142,20 @@ module.exports = function(codeCoverage, browserstack, watchMode, file) {
     // enable / disable watching file and executing tests whenever any file changes
     autoWatch: true,
 
-    reporters: ['mocha'],
+    reporters: ['mocha', 'coverage'],
+
     mochaReporter: {
       showDiff: true,
       output: 'minimal'
     },
 
+    coverageReporter: {
+      dir: 'build/coverage',
+      reporters: [
+        { type: 'html', subdir: 'karma-html' }
+      ]
+    },
+
```

Comment:
As a question, I noticed with this block of code added here (and potentially at the `reporters` variable above), we now seem to always create this `coverage` directory (such as when we run `gulp serve`).  Is this intended and needed?  Or can we alter the code some-what to only generate these folders and files when the `test-coverage` command is ran?

Reply:
@jsnellbaker Thanks for the feedback. That was a good catch, I shouldn't have gotten rid of `if(codeCoverage)` in `function setReporters`. I've added it back, and now it only generates coverage folder when we run the command, `gulp test-coverage`